apps/mull-ui/src/app/app.tsx

```diff
@@ -90,8 +90,8 @@ export const App = () => {
           />
           <Route exact path={ROUTES.TOKEN_REDIRECT} component={TokenRedirectPage} />
           <PrivateRoute exact path={ROUTES.CREATE_EVENT} component={CreateEventPage} />
-          <Route exact path={ROUTES.EVENT_BY_ID} children={<EventPage />} />
-          <PrivateRoute path={ROUTES.HOME}>
+          <Route exact path={ROUTES.EVENT_BY_ID} children={<EventPage prevPage="Discover" />} />
```

Comment:
💡 I'd say you can revert this change, we can't know for sure that the discover page was the last page for the user